pkg/generated/openapi/BUILD

```diff
@@ -5,6 +5,10 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 gen_openapi(
     outs = ["zz_generated.openapi.go"],
+    exclude_pkgs = [
+        "staging/src/k8s.io/code-generator",
+        "staging/src/k8s.io/sample-apiserver",
```

Comment:
did we include these packages before? I didn't see them in `pkg/generated/openapi` from HEAD

Reply:
the code generator one might be superfluous here because bazel ignores it via some other mechanisms. The list is a start only though, we will add apiextensions and kube-aggregator in a follow-up.